### ISSUE TITLE
History DB Auto-cleanup

### ISSUE THREAD

**Issue by kubevious** [kubevious/kubevious#8]
History DB grows infinitely. Would eventually cause out-of-space error on MySQL. Implement auto-cleanup capability. Cleanup based on time/disk space.

**Comment by sabper** [comment]
thank you. 
my kubevious mysql disk full..
how can i cleanup with manual?

**Comment by rubenhak** [comment]
> thank you.
> my kubevious mysql disk full..
> how can i cleanup with manual?

Could you share some deployment numbers (#deployments, #pods, #nodes)? How long have you been using it?

I'd recommend increasing the volume size for some time until the proper fix is ready. There should be a persistent volume **data-kubevious-mysql-0** in namespace **kubevious**. 10GB is definitely not enough. This can help to increase the volume size: [https://dev.to/bzon/resizing-persistent-volumes-in-kubernetes-like-magic-4f96](https://dev.to/bzon/resizing-persistent-volumes-in-kubernetes-like-magic-4f96). You will need to restart the **kubevious-mysql-0** pod to accept the larger volume size

**Comment by sabper** [comment]
thank you for reply.

i have 31 deployment, 75 pod, 8 nodes.
i have been use for 5days.

i consider increasing volume size for kuvevious mysql.

**Comment by rubenhak** [comment]
@sabper, hi, we made a significant redesign to the database. Now it should consume much less space. Please follow instructions here: [https://github.com/kubevious/helm](https://github.com/kubevious/helm).

Any further upgrades would be through helm releases, but this time please manually cleanup the namespace before following instructions above:
```sh
kubectl delete namespace kubevious
kubectl delete clusterrole kubevious
kubectl delete clusterrolebinding kubevious
```

We are still working on db-cleanup, but this should help a lot until rotation is implemented.

**Comment by kubevious** [comment]
Resolved in [v0.7](https://github.com/kubevious/kubevious/releases/tag/v0.7).
